server/models/ClubOverviewModel.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const clubOverviewSchema = new Schema({
+    name: { type: String, required: true },
+    description: { type: String, required: true },
+    imageUrl: { type: String, default: '' },
+    memberRange: { type: String, required: true },
+    acceptingMembers: { type: Boolean, required: true },
+    applicationRequired: { type: Boolean, required: true },
+    tags: {
+        type: [String],
+        required: true
+    },
+
+});
+
+const clubOverview = mongoose.model('ClubOverview', clubOverviewSchema);
+
+module.exports = clubOverview;
```

Comment:
the reason why i think this schema shouldn't exist is: 
1. This will be creating a duplicate copy for the same data, which is not ideal (especially since our spatial constraints are more stringent than time constraints as we only have ~500MB storage), and it also causes consistency issues. 
2. We need to decouple the db layout from the specific frontend use cases as soon as possible; the db shouldn't consider how the frontend will use the data specifically (otherwise we'll have to create a schema for every different page layout). The performance would be comparable with or without clubOverviewSchema, since the two schemas store an equal number of rows of data and we'll have to scan through them anyway (the specific number of fields of the data don't matter that much, especially since right now we only have so few rows for each object).
Please let me know your thoughts on this!